README.md

```
@@ -74,40 +87,80 @@ pnpm db:push     # Apply migrations to the database
 - `POST /preferences` - Update user email preferences
 - `GET /default-preferences` - Get default email preferences
 
 ### Email Sending
+
 - `POST /resend` - Resend the last sent email
 
+### Cron Jobs
+
+- `GET /cron/usage-reports` - Trigger usage reports cron job (protected by CRON_SECRET). It is expected in the Authorization Header as "Bearer CRON_SECRET"
+
 ### Admin
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)